### PR TITLE
feat: first draw of Chat Room with Websockets

### PR DESCRIPTION
**Pull request by picolloo**

### CODE REVIEW

File: issue98-websockets/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/server.js",
+    "dev": "nodemon src/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/picolloo/CurriculumExpansion.git"
+  },
+  "author": "Lucas Picollo",
```
Comment (thecodingaviator) [inline issue98-websockets/package.json]: I'm not sure if we want to leave the rights with @picolloo or @freeCodeCamp would take over it.

cc: @Ethan-Arrowood

File: issue98-websockets/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "version": "1.0.0",
+  "description": "Building a chat using WebSockets",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/server.js",
+    "dev": "nodemon src/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/picolloo/CurriculumExpansion.git"
```
Comment (thecodingaviator) [inline issue98-websockets/package.json]:
```suggestion
    "url": "git+https://github.com/freeCodeCamp/CurriculumExpansion.git"
```
I guess we'd be updating all these URLs

File: issue98-websockets/package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "author": "Lucas Picollo",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/picolloo/CurriculumExpansion/issues"
```
Comment (thecodingaviator) [inline issue98-websockets/package.json]:
```suggestion
    "url": "https://github.com/freeCodeCamp/CurriculumExpansion/issues"
```

File: issue98-websockets/package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "homepage": "https://github.com/picolloo/CurriculumExpansion#readme",
```
Comment (thecodingaviator) [inline issue98-websockets/package.json]:
```suggestion
  "homepage": "https://github.com/freeCodeCamp/CurriculumExpansion#readme",
```

**Comment by scissorsneedfoodtoo** [comment]
@picolloo, sorry, but I just noticed that this PR is coming from your master branch. Could you create a new branch and work off of that? Just let me know if I can help with that.

**Comment by picolloo** [comment]
@scissorsneedfoodtoo I've created a develop branch

**Comment by scissorsneedfoodtoo** [comment]
@picolloo, okay, I'll close this PR for now. Please continue working on your project and when you break it down into steps, submit another PR from your development branch. Thank you!

**Pull request closed without merging**